config/protractor/protractor-dev.conf.js

```diff
@@ -9,6 +9,82 @@ const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 const common = require('../../e2e/shared/common');
 const commonConfig = require('./protractor.conf');
 
+/**
+ * Clones the skyux-template from a specific branch.
+ */
+function cloneTemplate() {
+  const url = 'https://github.com/blackbaud/skyux-template';
+  const branch = 'builder-dev';
+
+  return common.exec(`git`, [
+    `clone`,
+    `-b`,
+    branch,
+    `--single-branch`,
+    url,
+    common.tmp
+  ]);
+}
+
+/**
+ * Copies current builder dependencies to cloned template.
+ */
+function mergeDependencies() {
+  const builderJson = fs.readJSONSync('package.json');
+  const templateFilename = path.resolve(common.tmp, 'package.json');
+  const templateJson = fs.readJSONSync(templateFilename);
+
+  for (let key in builderJson.dependencies) {
+    if (builderJson.dependencies.hasOwnProperty(key)) {
```

Comment:
Non-blocking, but `Object.keys(` would let you skip this `if` check...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys

```
Object.keys(builderJson.dependencies).forEach((key) => {
  templateJson.dependencies[key] = builderJson.dependencies[key];
});
```

Reply:
Great suggestion.